rocketpy/Rocket.py

```diff
@@ -267,6 +269,7 @@ def __init__(
         # Calculate dynamic inertial quantities
         self.evaluateReducedMass()
         self.evaluateTotalMass()
+        self.thrustToWeight = self.motor.thrust/(9.80665*self.totalMass)
```

Comment:
```suggestion
        self.thrustToWeight = self.motor.thrust/(9.80665*self.totalMass)
        self.thrustToWeight.setInputs('Time (s)')
        self.thrustToWeight.setOutputs('Thrust/Weight')
```

Reply:
Ty so much, now it is working!!!!! 

See an example:

![image](https://user-images.githubusercontent.com/63590233/89102302-309cbc00-d3de-11ea-91ce-bd0fc597ba78.png)